.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -84,6 +84,9 @@ As we resolve various outstanding known issues and implement new functionality w
 We should check and update if necessary:
 
 - [ ] The "Known Issues" section
+- [ ] The "Build and Run Instructions" section. Check if any new dependencies were introduced and
+      list them if needed; one possible approach is to check for changes in the `Dockerfile`
+      since the last tag: `git diff <previous-release-tag> docker/Dockerfile`.
 
 to ensure that any items that are resolved in the latest release are no longer listed in the README.
```

Comment:
Why did you remove the "Known Issues" item? Here's a suggestion that keeps that item, and appends the sentence that was below the list to it.
```suggestion
- [ ] The "Known Issues" section to ensure that any items that are resolved in the latest release are no longer listed in the README.
- [ ] The "Build and Run Instructions" section. Check if any new dependencies were introduced and
      list them if needed; one possible approach is to check for changes in the `Dockerfile`
      since the last tag: `git diff <previous-release-tag> docker/Dockerfile`.
```

Reply:
I didn't remove it? :sweat_smile: 

But your suggestion is much better, I'll commit it

Reply:
Oh! :D I'm sorry, I confused the "-" sign indicating a list item with the diff's "removed" indicator. My colorblindness doesn't help with this business.

Reply:
No problem! I figured something like that had happened :laughing: